cmd/promtool/unittest.go

```diff
@@ -18,19 +18,20 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"os"
-	"path/filepath"
-	"sort"
-	"strconv"
-	"strings"
-	"time"
-
 	"github.com/go-kit/log"
 	"github.com/google/go-cmp/cmp"
 	"github.com/grafana/regexp"
 	"github.com/nsf/jsondiff"
 	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/util/junitxml"
 	"gopkg.in/yaml.v2"
+	"io"
```

Comment:
Linter complains about the ordering of imports, see https://github.com/prometheus/prometheus/actions/runs/9920970887/job/27408406132?pr=14461